sklearn/tests/test_isotonic.py

```diff
@@ -27,6 +27,14 @@ def test_permutation_invariance():
     assert_array_equal(y_transformed, y_transformed_s)
 
 
+def test_check_increasing_small_number_of_samples():
+    x = [0, 1, 2]
+    y = [1, 1.1, 1.05]
+
+    is_increasing = assert_no_warnings(check_increasing, x, y)
+    assert_true(is_increasing)
```

Comment:
I am not sure what the right fix is for this problem but I would say that `check_increasing([0, 1, 2], [1, 1.1, 1.05])` should return False and not True as seemed to be the case in this test.